### PR TITLE
Course

### PR DESCRIPTION
**Pull request by lrmi0808**
Course Model

### CODE REVIEW

**Comment by lrmi0808** [comment]
Ready

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "All users") %>
+  <h1>All users</h1>
+
+  <%= will_paginate %>
+
+  <ul class="users">
+      <%= render @users %>
```
Comment (huongnt-2545) [inline app/views/users/index.html.erb:7]: wrong indent

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,11 +1,65 @@
...
+    </aside>
+  </div>
+
+  <div>
+    <p>
+      <% if current_user? @user %>
+        <b><%= t(".register_course") %>: </b>
+        <%= form_for @register do |f| %>
+          <%= f.hidden_field :user_id, :value => current_user.id %>
+          <%= f.collection_select :course_id, Course.all, :id, :name, { include_blank: "" } %>
```
[the remark this answers is not in the document]
Comment (lrmi0808) [reply app/views/users/show.html.erb:21]: May I know how to do this?

**Comment by lrmi0808** [comment]
Ready

File: app/helpers/courses_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module CoursesHelper
```
Comment (anlvq-1058) [inline app/helpers/courses_helper.rb:1]: delete this file if you not use

File: app/helpers/registers_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module RegistersHelper
```
Comment (anlvq-1058) [inline app/helpers/registers_helper.rb:1]: delete this file if you not use

File: app/views/users/show.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (anlvq-1058) [reply app/views/users/show.html.erb:21]: You can find all the couse user register by ids = current_user.couse_ids.
Then use Couse.not.where id: ids

**Pull request merged**